### ISSUE TITLE
support for android

### ISSUE THREAD

**Issue by pschiffe** [pomerium/cli#368]
Hello. There's a great app for running SSH on Android - https://juicessh.com/
However, if the SSH is behind Pomerium TCP route, AFAIK there's no way currently to connect to it from Android phone or tablet.
Having SSH access on phone is usually for emergencies only, but tablets today are very capable and can be used as a laptop replacement.

**Comment by wasaga** [comment]
https://github.com/brian-mcnamara/android-pomerium-bridge

**Comment by desimone** [comment]
@ pschiffe was the link @wasaga  what you were looking for?

**Comment by pschiffe** [comment]
Not really, as I need to compile that myself? I'm not sure, there's no documentation..

I'm looking for something I could download from the play store.

**Comment by desimone** [comment]
Got it. The best way to handle this would be to wrap `ProxyCommand`. I'm not sure the best way to do this in the android ecosystem but it's native to OpenSSH.  

https://www.pomerium.com/docs/capabilities/tcp/examples/ssh.html#always-tunnel-through-pomerium

**Comment by pschiffe** [comment]
I think JuiceSSH could handle `ProxyCommand` directive in the SSH connection setting, alternatively it supports "snippets" - run custom code before creating the connection.
However, the part that is missing is to run `pomerium-cli` on the android device.

I can also imagine, and maybe it would be even better, to have pomerium app for android similar to pomerium desktop - a graphical application to create connections from localhost to pomerium routes. That could work for ssh as well as databases.

**Comment by desimone** [comment]
Sorry, I'm not super familiar with the android ecosystem (in theory, it should be as simple as having `pomerium-cli` in `/bin` somewhere?) but in theory pomerium-cli can probably run on android architecture (we support bunch of different flavors including [arm](https://github.com/pomerium/cli/releases/tag/v0.22.0 ) and I don't with little modification on the [build target](
https://github.com/golang/go/wiki/Mobile#building-and-deploying-to-android). 

Though we don't have the resources (or devices) to do this type of work currently, either an app or updated build target for android and usage instructions would be welcome contributions from the community!

- https://pkg.go.dev/golang.org/x/mobile/cmd/gomobile
